src/server/req_quejob.c

```diff
@@ -255,7 +255,7 @@ validate_perm_res_in_select(char *val, int val_exist)
 static int
 generate_objid(char *idbuf, char *clusterid, int objtype, char resv_char)
 {
-	if (idbuf == NULL || server_name == NULL || clusterid == NULL)
+	if (idbuf == NULL || clusterid == NULL)
```

Comment:
The server_name can never be NULL? Or perhaps we do a server_name[0] == '/0' ?

Reply:
well that was the compiler error.
but checking for server[0] is a good idea